Doc/faq/design.rst

```diff
@@ -62,7 +62,7 @@ and think it is a bug in Python.  It's not.  This has little to do with Python,
 and much more to do with how the underlying platform handles floating-point
 numbers.
 
-The :class:`float` type in CPython uses a C ``double`` for storage.  A
+The :class:`float` type in CPython uses a C :c:type:`double` for storage.  A
```

Comment:
If I recall, I believe that regular literals (double backticks) are how you've handled this sort of thing elsewhere.

Reply:
Doing a quick search through the repo I'm seeing more (23 vs 5) in the form of `` :c:type:`double` ``, so keeping it would be more in line with what's currently being done.

Reply:
Okay, I see. Still, it is strange that it is formatted as prose rather than as a literal, as we'd (and I assume the original authors) would expect—perhaps something changed in the theme, Sphinx or the configuration that resulted in that not happening as expected.